src/homepageExperience/containers/GoWizard.tsx

```diff
@@ -70,7 +70,14 @@ export class GoWizard extends PureComponent<null, State> {
         ),
       },
       () => {
-        event('firstMile.goWizard.next.clicked')
+        event(
+          'firstMile.goWizard.next.clicked',
+          {clickType: 'next'},
```

Comment:
Do we need to pass `clickType`? given that it is part of `'firstMile.goWizard.next.clicked'` 

Rest looks good!

Reply:
Since it's param 2 out of 3 we'll need to pass something now, but the default value in the code is just an empty `object`, so we could do that if it's not needed!

Reply:
>Since it's param 2 out of 3 we'll need to pass something now

it's okay to send an empty object to the second argument. [Example here](https://github.com/influxdata/ui/blob/eb134fded2a37b637d7c39f8067d966cc6350c3d/src/timeMachine/actions/queries.ts#L277)

```ts
event(
  'firstMile.goWizard.next.clicked',
  {},
  {
    clickedButtonAtStep: this.state.currentStep - 1,
    currentStep: this.state.currentStep,
  }
)
```

Reply:
Per above, removed the tags, and am now passing in an empty object. One thing I noticed when re-testing: even if the `streamEvents` flag is turned on, it won't console.log event values unless the event has a tag. Fixed the `if` blocks so that streamEvents still provides console.logs if the event has values but no tag.

Let's wait on @Luckette to make sure this format looks good to her.

https://user-images.githubusercontent.com/91283923/169574222-89c529b1-f8ff-4437-9b28-653e131458e1.mov